File: code-examples/hashing/hashing.go
package main

import (
    "crypto/sha256"
    "encoding/hex"
    "fmt"
)

func main() {
    stringToHash := "Mensagem que vai ser sumarizada"

    hashObject := sha256.New()             // criando objeto que vai realizar o hash da mensagem (em SHA-256)
    hashObject.Write([]byte(stringToHash)) // Hashing

    hexHash := hex.EncodeToString(hashObject.Sum(nil)) // resgatando o valor do hash em hexadecimal

    fmt.Printf("O hash de '%s' em hexadecimal utilizando SHA-256 é: %s\n", stringToHash, hexHash)
}

File: code-examples/hashing/hashing.py
import hashlib

string_to_hash = "Mensagem que vai ser sumarizada"

hash_object = hashlib.sha256()  # criando objeto que vai realizar o hash da mensagem (em SHA-256)
hash_object.update(string_to_hash.encode())  # Hashing

hex_hash = hash_object.hexdigest()  # resgatando o valor do hash em hexadecimal

print(f"O hash de '{string_to_hash}' em hexadecimal utilizando SHA-256 é: {hex_hash}")